notebooks/D_CNN_Inference_and_Embedding.ipynb

```diff
@@ -268,7 +267,7 @@
     "        plt.title('Image {}'.format(image_num+1))\n",
     "        plt.yticks([])\n",
     "        plt.xticks([])\n",
-    "        plt.xlabel(LABELS[np.argmax(test_predictions[image_num])])\n",
+    "        plt.xlabel(list(states)[np.argmax(test_predictions[image_num])])\n",
```

Comment:
might be better to make a variable and reuse it:
```python
LABELS = list(states)
```

Reply:
done! I've updated the above PR description according to the new changes